CHANGELOG.md

```diff
@@ -152,7 +153,8 @@
 - PR #4066 Fixed mismatch with dtype enums
 - PR #4078 Fix joins for when column_in_common input parameter is empty
 - PR #4080 Fix multi-index dask test with sort issue
-- PR #4084 Update Java for removal of CATEGORY type
+- PR #4084 Update Java for removal of CATEGORY type partitioning
+- Run cross validation benchmark based o
```

Comment:
This seems like a typo.

Reply:
Oops. Not sure how this was copied. Since I had changed only this file, hadn't self  reviewed after commiting.